doc/user_guide/user_guide.md

```diff
@@ -330,7 +335,7 @@ In integration tests you usually want reproducible test cases. This is why the s
 
 In rare cases you might want more control over that process, for example if you plan bulk-upload of a large number of small files and want to shift the check to the end of that operation.
 
-For those special occasions there is an overloaded method `uploadFile(source, destination, blocking-flag)` where you can choose to upload in non-blocking fashion. 
+For those special occasions there is an overloaded method `uploadFile(source, destination, blocking-flag)` where you can choose to upload in non-blocking fashion.
```

Comment:
Where does all the trailing whitespace come from? :-D

Reply:
🤣